crowbar_framework/lib/crowbar/lock.rb

```diff
@@ -16,33 +16,14 @@
 #
 
 module Crowbar
-  class Lock
-    attr_accessor :logger, :path, :name
-    attr_reader :path, :name, :file
+  module Lock
+    autoload :Base,
```

Comment:
Our current workaround is https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/config/application.rb#L22-L24 and the motivation is simply being more in line with rails conventions.